src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -188,10 +188,10 @@ export class StackDetailsComponent implements OnInit {
     const url: string = this.similarStacks[0].uri;
     this.recoArray[this.currentIndex]['rows'] = [];
     this.recoArray[this.currentIndex]['url'] = url;
-    for (var component in components) {
+    for (let component of components) {
```

Comment:
@joshuawilson Yes. Joshua. It has to be for in - in this case. As for of directly returns the iterable object in the variable. Will fix it